### PR TITLE
changet language tabs

### PR DESCRIPTION
**Pull request by RevylaV**
Знімок екрана 2021-03-09 о 19 41 40

### CODE REVIEW

**Comment by Haytham1337** [comment]
fixed codesmells please)

File: src/components/forms/material-form/material-form.js [hunk 1]
```diff
@@ -113,44 +113,17 @@ function MaterialForm({ material, id }) {
...
-          <div className={styles.inputError}>
-            {errors[`${lang}Description`]}
-          </div>
-        )}
-      </Paper>
-    </TabPanel>
-  ));
+  const inputs = [
+    { label: config.labels.material.name, name: 'name' },
+    { label: config.labels.material.description, name: 'description' }
```
Comment (AndriyLymych) [inline src/components/forms/material-form/material-form.js:118]: magic string

**Comment by AndriyLymych** [comment]
link your issue
image

**Comment by Haytham1337** [comment]
Do this to our style guide please.
image

**Comment by Kiyooooko** [comment]
fix codesmell pls

**Review by Haytham1337 — APPROVED** [review]

**Review by AndriyLymych — APPROVED** [review]

**Review by yuliiakucher — APPROVED** [review]

**Pull request merged**